server/schema/auth/updateUser.js

```diff
@@ -19,6 +19,15 @@ const schema = {
       maxLength: 10,
       pattern: '^[0-9]{10}$',
     },
+    department: {
+      type: 'number',
+    },
+    branch: {
+      type: 'number',
+    },
```

Comment:
Specify min and max for branch and department

Reply:
done